### DISCUSSION TITLE
2011/04/11/%E8%B1%86%E8%B1%86%E5%8F%88%E5%8E%BB%E4%BD%93%E6%A3%80%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#324] [Announcements]
# 2011/04/11/%E8%B1%86%E8%B1%86%E5%8F%88%E5%8E%BB%E4%BD%93%E6%A3%80%E4%BA%86

豆豆4个月，打疫苗顺便体检。现在身高71.2cm，体重8.5kg。

https://blog.qizhen.xyz/2011/04/11/%E8%B1%86%E8%B1%86%E5%8F%88%E5%8E%BB%E4%BD%93%E6%A3%80%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

lizzie
2011年04月13日 17:47 编辑
长得这么高了，羡慕ing

回复
